<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lblLeyenda1">
          <property name="text">
           <string>Enviar texto a nueva ventana:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtaEnviar"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnEnviar">
          <property name="text">
           <string>Enviar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="btnMiImagen">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCargarImagen">
          <property name="text">
           <string>Cargar imagen</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="chkModo">
          <property name="text">
           <string>Horizontal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCambiarModo">
          <property name="text">
           <string>Cambiar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="pbMiBarra">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QDateEdit" name="dtFecha"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Obtener</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QDial" name="dial"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnVerValor">
          <property name="text">
           <string>Ver valor</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="lblFecha"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUn_submenu"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>466</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHola">
    <property name="title">
     <string>hola</string>
    </property>
    <widget class="QMenu" name="menuOtroMenu">
     <property name="title">
      <string>otroMenu</string>
     </property>
     <addaction name="actionUn_submenu"/>
    </widget>
    <addaction name="actionMenu_1"/>
    <addaction name="menuOtroMenu"/>
   </widget>
   <addaction name="menuHola"/>
  </widget>
  <action name="actionMenu_1">
   <property name="text">
    <string>Menu 1</string>
   </property>
  </action>
  <action name="actionUn_submenu">
   <property name="icon">
    <iconset resource="imagenes.qrc">
     <normaloff>:/misRecursos/logo</normaloff>:/misRecursos/logo</iconset>
   </property>
   <property name="text">
    <string>un submenu</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
